docs/source/containers/index.md

```diff
@@ -0,0 +1,31 @@
+# Containers on HPC
+```{toctree}
+:hidden:
+:maxdepth: 4
+
+singularity
+```
+
+A container is a standard unit of software that packages up code and all its dependencies so the application runs quickly and reliably from one computing environment to another. A container image is a lightweight, standalone, executable package of software that includes everything needed to run an application: code, runtime, system tools, system libraries and settings. ([Reference](https://www.docker.com/resources/what-container/))
+
+Containers allow reproducibility and portability in scientific workflows. 
+
+They ensure that scientific experiments can be reproduced on any system, such as our laptop, our friend’s laptop, HPC as well as public cloud-based environments. Containers achieve this by encapsulating the entire software environment of the specific analysis or simulation. This eliminates the "it works on my machine" problem by providing a consistent runtime environment across various platforms.
+
+This consistency also allows for more collaboration, allowing researchers to share their work without encountering compatibility issues, thereby speeding up scientific workflows.
+
+Containers also simplify software dependency issues by isolating software and its dependencies from the underlying HPC environment. This reduces project conflicts and makes managing different software versions much more accessible. 
```

Comment:
This is all correct but some of it feels like it is the same content stated a slightly different way. This might be able to be reduced slightly but I think it is okay also as is too.